e2e/tests/insertPlugins.e2e.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable new-cap */
+/*global cy*/
+import {
+  TOOLBARS,
+  STATIC_TOOLBAR_BUTTONS_WITHOUT_EMBED,
+  STATIC_TOOLBAR_BUTTONS_MEDIA,
+  VIDEO_PLUGIN,
+  SOUND_CLOUD,
+  SOCIAL_EMBED,
+  // GIPHY_PLUGIN,
+} from '../cypress/dataHooks';
+import { DEFAULT_DESKTOP_BROWSERS } from './settings';
+import { useUploadConfig } from '../cypress/testAppConfig';
+
+const LINKS = {
+  YOUTUBE: 'https://www.youtube.com/watch?v=whbidPR4nVA',
+  SOUNDCLOUD: 'https://soundcloud.com/martingarrix/martin-garrix-animals-original',
+};
+
+const modalHandler = (INPUT_BUTTON, ADD_BUTTON, LINK) => {
+  cy.get(`[data-hook=${INPUT_BUTTON}]`).type(LINK);
+  cy.get(`[data-hook=${ADD_BUTTON}]`).click();
+  cy.wait(3000); //DO NOT REMOVE - fix flakiness
+};
+
+const additionalCommands = {
+  VIDEO: () => {
+    modalHandler(VIDEO_PLUGIN.INPUT, VIDEO_PLUGIN.ADD, LINKS.YOUTUBE);
```

Comment:
try to maybe do it like this
```suggestion
    modalHandler(LINKS.YOUTUBE);
```

Reply:
I'll keep it like that for now.
If we'll see in the future that it needs to be changed - I'll make a more generic implementation.